generators/component/templates/functions.php

```diff
@@ -2,49 +2,27 @@
 
 namespace Flynt\Components\<%= nameUpperCamelCase %>;
 
-use Flynt\Api;
 use Flynt\Utils\Options;
 
 add_filter('Flynt/addComponentData?name=<%= nameUpperCamelCase %>', function ($data) {
     return $data;
 });
 
-Api::registerFields('<%= nameUpperCamelCase %>', [
-    'layout' => [
-        'name' => '<%= nameLowerCamelCase %>',
-        'label' => '<%= namePrettySplit %>',
+function getACFLayout()
+{
+    return [
+        'name' => 'GridCards',
+        'label' => 'Grid: Cards',
         'sub_fields' => [
-            [
-                'label' => 'General',
-                'name' => 'generalTab',
-                'type' => 'tab',
-                'placement' => 'top',
-                'endpoint' => 0,
-            ],
-            [
-                'label' => 'Options',
-                'name' => 'optionsTab',
-                'type' => 'tab',
-                'placement' => 'top',
-                'endpoint' => 0,
-            ],
-            [
-                'label' => '',
-                'name' => 'options',
-                'type' => 'group',
-                'layout' => 'row',
-                'sub_fields' => [
-                    Api::loadFields('FieldVariables', 'theme'),
-                ],
-            ],
-        ],
-    ],
-]);
```

Comment:
why did you delete this? it does show our general approach of having a general and an options tab, and how to use the FieldVariables. The last point would have to be adjusted to the the syntax, though.

Reply:
@domtra I thought that the VScode snippets were meant for that and keeping it empty would save some time deleting the "standard" text but I see your point and I agree. I have made a couple adjustments and pushed to the branch.